site/dat/docs/Apiritif.md

```diff
@@ -475,6 +526,15 @@ For storing variables use the following actions:
 
 See documentation for `X` [here](#Locators).
 
+For storing the result of evaluation of a JavaScript expression use:
+
+- `storeEval(js_expr): var_eval`
+
+Note that the result of the evaluation will be converted to String data type.
+
+The following example will store the result of finding an element by id to the `el_present` variable:
+- `storeEval(document.getElementById("elem_id") !== undefined): el_present`
```

Comment:
In my point it isn't 'result of finding' (as result is webelement or mone). it's rather 'mark whether finding was successful'

Reply:
Good point...